src/amqproxy/client.cr

```diff
@@ -77,11 +77,11 @@ module AMQProxy
         end
       end
     rescue ex : IO::EOFError
-      Log.debug { "Disconnected" }
+      Log.debug(exception: ex) { "Disconnected" }
```

Comment:
Will this now also print the backtrace? (if so, not sure that's desirable)

Reply:
It would yes, which I agree is not ideal due to not all being on a single line. A possible compromise could be to have a custom log formatter to either output the trace on a single line, or only include first line of it, or possibly not at all when in the IO backend context if so desired.

This would at least make the IO based logs cleaner, while still exposing the exception information to other backends that can make better use of it.